extensions/machine-learning-services/src/common/utils.ts

```diff
@@ -176,3 +182,48 @@ export async function promptConfirm(message: string, apiWrapper: ApiWrapper): Pr
 
 	return choices[result.label] || false;
 }
+
+export function makeLinuxPath(filePath: string): string {
```

Comment:
there is no already existing util for this we can use?

Reply:
the extension code assumes it doesn't have access to other codes so the unless might get duplicated unless the functionalities are provided through the services

---
In reply to: [389191198](https://github.com/microsoft/azuredatastudio/pull/9450#discussion_r389191198) [](ancestors = 389191198)

Reply:
ok one more followup question on this one - you are using makeLinuxPath(os.homedir() - wont os.homedir() on Linux automatically give the path in Linux style? and if its client vs server path then there might be more differences than just / vs \ .. right? I am missing something here...

Reply:
no if you run in Windows but mlflow client works with linux paths only even in Windows

---
In reply to: [389919284](https://github.com/microsoft/azuredatastudio/pull/9450#discussion_r389919284) [](ancestors = 389919284)